2021-07-01 11:16:13,517:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:16:24,626:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:16:35,268:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:16:45,715:	INFO:	Hc.py:__hcUpdateDisconnectStToDb():113	Update cloud disconnect status to db
2021-07-01 11:16:45,718:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:16:56,366:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:17:06,649:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:17:13,554:	ERROR:	Mqtt.py:_connect():81	Exception in connect to mqtt: timed out
2021-07-01 11:17:17,096:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:17:18,825:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {
  "CMD": "GROUP",
  "DATA": {
    "GROUP_ID": "b9bc79b8-cdc5-44a7-be95-78c3341791cb",
    "PROPERTIES": [
      {
        "ID": 0,
        "VALUE": 1
      }
    ]
  }
}
2021-07-01 11:17:18,825:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:19,026:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:19,026:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:19,227:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:19,227:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:19,428:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:19,428:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:19,528:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:19,528:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:19,729:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:19,730:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:20,131:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:20,131:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:20,733:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:20,733:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:21,135:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:21,135:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:21,536:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:21,536:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:21,938:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:21,938:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:22,139:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:22,139:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:24,848:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:24,849:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:25,150:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:25,150:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:25,250:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:25,250:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:25,552:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:25,552:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:25,652:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:25,652:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:25,953:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:25,953:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:26,355:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:26,355:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:26,556:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:26,556:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:26,857:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:26,858:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:26,958:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:26,958:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:27,260:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:27,260:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:27,549:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:17:27,663:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: {"CMD":"DEVICE","DATA":{"DEVICE_ID":"268f9df6-95df-c43c-bf52-bb4b2dac442e","PROPERTIES":[{"ID":0,"VALUE":1},{"ID":1,"VALUE":100},{"ID":2,"VALUE":29}]}}
2021-07-01 11:17:27,663:	ERROR:	Hc.py:__mqttHandlerCmdConnectToCloud():188	mqtt data receiver invalid
2021-07-01 11:17:27,748:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:17:27,749:	INFO:	Hc.py:__hcUpdateReconnectStToDb():103	Update cloud reconnect status to db
2021-07-01 11:19:55,579:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:19:56,875:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:20:06,877:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:20:07,508:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:20:17,508:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:20:17,815:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:20:27,817:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:20:28,616:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:20:38,618:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:20:41,252:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:20:51,254:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:20:59,664:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:21:56,438:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:21:58,158:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:22:08,159:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:22:10,131:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:22:20,132:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:22:20,528:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:22:30,529:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:22:31,455:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:22:41,456:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:22:41,867:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:22:51,868:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:22:52,672:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:23:26,095:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:23:36,102:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:23:46,105:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:23:56,107:	INFO:	Hc.py:__hcUpdateDisconnectStToDb():113	Update cloud disconnect status to db
2021-07-01 11:23:56,158:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:24:06,161:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:24:16,163:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:24:26,165:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:24:36,168:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:24:37,310:	DEBUG:	Hc.py:__mqttHandlerTopicHcControl():154	mqtt data receive from topic HC.CONTROL: { "CMD":"HC_CONNECT_TO_CLOUD", "DATA":{ "END_USER_PROFILE_ID":10103,"REFRESH_TOKEN":"OGj2Lo8elXRxCiOit9i2lmZlHlOqA5KCoQ8NEAkmcU94F8zMFmpED3SQzd7OJDOvkJd0yWBPjP32y6fuSCvJCXlCbvWVrktDWe67YvMcjHnZCzOR1mu3+c6lKNGHWvfvtDLJTpWZQIyWYqx/+l3agx0ULYckAvV5K3gKfgVNem/JOP57IVEWLdSdMwuMIQwIDh0MgUiqpbFEse0kVWPr1T9CChTerHDDn5sMBvWh4ug="}}
2021-07-01 11:27:38,086:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:28:40,517:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:28:41,364:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:28:56,530:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:28:58,846:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 11:29:24,676:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 11:29:26,157:	ERROR:	Signalr.py:ReConnect():97	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:31:55,604:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:32:06,839:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:32:18,621:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:32:31,545:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:33:02,600:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:33:03,597:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:33:08,601:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:33:13,608:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:33:13,634:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:33:18,612:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:33:23,616:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:33:23,638:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:33:28,621:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:33:33,628:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:33:33,641:	INFO:	Hc.py:__hcUpdateDisconnectStToDb():111	Update cloud disconnect status to db
2021-07-01 12:36:05,652:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:36:06,647:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:36:11,651:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:36:15,660:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:36:16,659:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:36:21,664:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:36:25,664:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:36:26,668:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:36:31,676:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:36:35,668:	INFO:	Hc.py:__hcUpdateDisconnectStToDb():111	Update cloud disconnect status to db
2021-07-01 12:36:35,695:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:36:36,680:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:40:48,730:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:40:49,732:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:40:54,738:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:40:58,737:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:40:59,741:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:41:04,748:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:41:08,741:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:41:09,756:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:41:14,760:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
2021-07-01 12:41:18,745:	INFO:	Hc.py:__hcUpdateDisconnectStToDb():113	Update cloud disconnect status to db
2021-07-01 12:41:18,794:	INFO:	Hc.py:__HcCheckConnectWithCloud():46	Hc send heardbeat to cloud
2021-07-01 12:41:19,768:	ERROR:	Signalr.py:__startConnect():56	Exception when connect with signalr server: can only concatenate str (not "NoneType") to str
